src/components/FeedbackCard.tsx

```diff
@@ -40,26 +40,6 @@ const ButtonContainer = styled.div`
   gap: 1rem;
 `
 
-const StyledButtonSecondary = styled(ButtonSecondary)`
-  display: flex;
-  gap: 0.5rem;
-  color: ${({ theme }) => theme.colors.primary};
-  border-color: ${({ theme }) => theme.colors.primary};
-  line-height: 140%;
-  vertical-align: middle;
-  svg {
-    height: 1.5rem;
-    &.flip {
-      transform: scaleY(-1);
```

Comment:
I believe we still need this transform: https://discord.com/channels/714888181740339261/938069231570923550/1019727618666410034